packages/analytics/src/Analytics.ts

```diff
@@ -32,11 +35,21 @@ import type {
 } from './types/analytics.types.js';
 import type { Storage } from './utils/types/index.js';
 
+// We use a require here to avoid typescript complaining of `package.json` is not
+// under rootDir that we would get if we used an import. Typescript apparently ignores
+// requires.
+const {
+  name: PACKAGE_NAME,
```

Comment:
These are already being exported from the `utils/constants.js` module. You can update the import you already have with that.